src/store/modules/aeternity.js

```diff
@@ -12,6 +12,7 @@ import FUNGIBLE_TOKEN_CONTRACT from 'aeternity-fungible-token/FungibleTokenFullI
 import { BigNumber } from 'bignumber.js';
 import { IS_MOBILE_DEVICE } from '../../utils';
 
+// todo: getters
```

Comment:
Can you remove this comment?

If the PR is still work in progress you can mark it as draft.